src/testrun.rs

```diff
@@ -92,6 +92,8 @@ pub struct Testrun {
     pub failure_message: Option<String>,
     #[pyo3(get, set)]
     pub filename: Option<String>,
+    #[pyo3(get, set)]
+    pub build_url: Option<String>,
```

Comment:
@joseph-sentry I believe worker gets the `build_url` from the `upload`. I don't think this is available for the CLI, but I just want to check. Code that gets it: https://github.com/codecov/worker/blob/85f31744eebbc96c564ca4ca06faf5a3e1c1ed3f/tasks/test_results_finisher.py#L231C1-L231C59